modules/statistics/server/jobs/daily-statistics.server.job.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/**
+ * Task that collects daily statistics and sends them to InfluxDB
+ */
+
+/**
+ * Module dependencies.
+ */
+var path = require('path'),
+    config = require(path.resolve('./config/config')),
+    influxService = require(path.resolve('./modules/core/server/services/influx.server.service')),
+    statistics = require(path.resolve('./modules/statistics/server/controllers/statistics.server.controller'));
+
+module.exports = function(job, agendaDone) {
+
+  statistics.getUsersCount(function(err, count) {
+
+    if (err) {
+      console.error('Daily statistics: failed fetching user count.');
+      console.error(err);
+      return agendaDone(err);
+    }
+
+    // Save to influx here
+    influxService.writePoint('members', count, { members: 'members' }, function(err) { // removed result from callback function because of ESLint complains
+      if (err) {
+        // InfluxDB was not enabled, don't scream out error
+        if (config.influxdb || !config.influxdb.enabled) {
+          console.log('Skipped storing daily statistics: no InfluxDB configured.');
+          // Let `agendaDone()` succeed
+          err = null;
+        } else {
+          console.error('Daily statistics: failed writing to InfluxDB.');
+          console.error(err);
+        }
+      }
+      agendaDone(err);
+    });
+  });
+
+};
```

Comment:
For this case the simple count would make it much easier to graph. It is just simply a graph of the values that are regularly shoved into influxdb. One value per interval period.

To make the graph from +1/-1 it would need to:
- group them by time buckets `group by time(1d)`
- sum the values `sum(<val>)`
- ...then accumulate the result from the beginning of time to get the actual count

the last step is the tricky one as it involves all of time. it's unnecessarily complicated way of storing the information if it was just for that use case.

storing the events is useful only for other query types:
- you can pick an arbitrary time period and ask how many users signed up in that period
- you can filter by any of the tag values (this is the big win I think, for example you could store the "source" of the user signup if available, etc...).

perhaps playing with example/seed/historic data in influxdb/grafana would help.